### PR TITLE
Changes to Alteration of Rules

### PR DESCRIPTION
**Pull request by bradleysigma**
* Reduce the notice period of a constitutional amendment to match the notice period of a general meeting.

### CODE REVIEW

File: constitution.md [hunk 1]
```diff
@@ -238,23 +238,16 @@
...
+### 21 Alteration of Rules  
+21.1 These rules may be amended or added to from time to time by a special resolution carried at any General Meeting.  
+21.2 However, any amendment or addition is valid only if it is registered by and approved by the Clubs and Societies Administration Committee.  
+21.3 A special resolution to amend or add to these rules is passed at a meeting if:  
+* at least two-thirds (2/3) of the members of the Society who vote in person or by proxy at the meeting vote in favour of the resolution; and  
+* any additional requirement of the Constitution relating to the passage of a special resolution have been met.  
+  
+21.4 A resolution is not considered a special resolution unless at least fourteen (14) days notice has been given to all members of the Society specifying that the resolution is to be considered a special resolution.  
+21.5 A declaration by the chairperson of the meeting that the special resolution has been carried is conclusive proof of that fact unless is poll is demanded by three (3) or more members present at the General meeting.
```
Comment (jsutton101) [inline constitution.md]: Typo - should be "unless a poll"

**Comment by jsutton101** [comment]
Is there a reason for not preserving the current requirement for 75% of votes?

**Comment by bradleysigma** [comment]
> 
> 
> Is there a reason for not preserving the current requirement for 75% of votes?

Because I copied this from a different constitution without noticing the change.

**Comment by bradleysigma** [comment]
Approved at AGM 8th Oct 2019.

**Pull request merged**